pkg/landscaper/deployitem/deployitem.go

```diff
@@ -0,0 +1,46 @@
+// SPDX-FileCopyrightText: 2020 SAP SE or an SAP affiliate company and Gardener contributors.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package deployitem
+
+import (
+	"context"
+
+	"k8s.io/apimachinery/pkg/types"
+
+	lsv1alpha1 "github.com/gardener/landscaper/apis/core/v1alpha1"
+	lsv1alpha1helper "github.com/gardener/landscaper/apis/core/v1alpha1/helper"
+	"github.com/gardener/landscaper/pkg/landscaper/operation"
+)
+
+// Operation contains all deployitem operations
+type Operation struct {
+	operation.Interface
```

Comment:
do you really need the whole operation interface?

Reply:
Probably not, but I thought it would make it easier to understand if this was handled the same way as in the other reconcile loops. Also, this way it is easier to extend if we need some additional modifications in future. But I can also get rid of it.
WDYT?

Reply:
as you only need the client I think there is no benefit using the whole operation.
If we need other methods in the future we can reconsider using the whole installation operation.